plat/hikey/hikey_def.h

```diff
@@ -42,17 +42,23 @@
 #define XG2RAM0_BASE			0xF9800000
 #define XG2RAM0_SIZE			0x00400000
 
+#define PLAT_TRUSTED_SRAM_ID	0
+#define PLAT_DRAM_ID		1
+
 /*
  * DRAM at 0x0000_0000 is divided in two regions:
  *   - Secure DRAM (default is the top 16MB except for the last 2MB, which are
  *     used by the SCP for DDR retraining)
  *   - Non-Secure DRAM (remaining DRAM starting at DRAM_BASE)
  */
-#define DRAM_SEC_SIZE			0x01000000
+#define DRAM_SCP_SIZE		0x00000000
```

Comment:
I was trying to prepare in advance and put in a 'placeholder' for PR6 but since it has to be refactored into smaller patches, maybe we should just get rid of SCP for now and let Leo add his own definitions later?